contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -56,6 +56,14 @@ LANGUAGE plpgsql;
  * final behaviour.
  */
 
+-- 3.1 introduces a change to how table types are stored in pg_type. While MVU forces the catalog
```

Comment:
Please add the same in 2.3.0--2.4.0 upgrade script because we'd like to see all upgrade scripts in the latest branch.